docs/api/middleware.rst

```diff
@@ -71,6 +71,11 @@ Falcon's middleware interface is defined as follows:
     component modifies ``req.path`` in its *process_request* method,
     the framework will use the modified value to route the request.
 
+    For example::
+
+        # Route requests based on the host header
+        req.path = '/' + req.host + '/' + req.path
```

Comment:
`Request.path` normally already starts with a `'/'`, so the `'/' + req.path` part of the expression would yield a double slash, which may confuse users.

Reply:
Ah shoot, wasn't thinking straight. I blame the flu. 😉  

Fixed.

Reply:
Looks good now :+1: